actions/publish-github-pages/action.yml

```diff
@@ -1,10 +1,17 @@
 name: 'Publish GitHub Pages'
 description: 'Publish the Hugo-based CloudPosse docs website to GitHub Pages.'
 runs:
-#  using: 'docker'
-#  image: 'Dockerfile'
   using: 'composite'
   steps:
-    - run: ${{ github.action_path }}/entrypoint.sh
+#    - uses: actions/setup-python@v2
+#      with:
+#        python-version: '3.x'
+    - run: |
+        export STAGING_DIR=/tmp/staging/
+        export GITHUB_PAGES_PUSH_PATH=$(pwd)/${GITHUB_PAGES_DIRECTORY}
+        pip install GitPython
+        python ${{ github.action_path }}/entrypoint.py
+        cd $STAGING_DIR && docker build -t cloudposse/docs .
+        cd $STAGING_DIR && make release && make real-clean hugo/build
+        cp -r ${STAGING_DIR}/${HUGO_PUBLISH_DIR} ${GITHUB_PAGES_PUSH_PATH}
```

Comment:
These belong in separate steps, no? Rather than overloading one step. Breaking it down into steps makes it easier to see where things fail.